Script/char.dat

```diff
@@ -5336,8 +6063,24 @@ darkmage
   CanRead = true;
   CanBeGenerated = true;
   ClassStates = INFRA_VISION;
-  HostileReplies == "@Dd yells: \"Prepare to meet your maker!\"";
-  FriendlyReplies == "\"Death awaits our foes.\"";
+  HostileReplies =
+  {
+    3,
+    "\"Avada Kedavra!\"",
+    "@Dd screams: \"May the Darkness swallow you!\"",
+    "@Dd yells: \"Prepare to meet your maker!\"";
+  }
+  FriendlyReplies =
+  {
+    7,
+    "\"Don't fear the Darkness, my friend.\"",
+    "\"Light thinks it travels faster than anything but it is wrong. No matter how fast Light travels, it finds the Darkness has always got there first, and is waiting for it.\"",
+    "\"Everything looks better when it's dead and on fire!\"",
+    "\"My motto? If it's not on fire now, it will be soon.\"",
+    "\"What hurts you, blesses you. Darkness is your candle.\"",
+    "@Dd cackles madly: \"Build a man a fire, and he'll be warm for a day. Set a man on fire, and he'll be warm for the rest of his life.\"",
+    "\"Death awaits our foes.\"";
+  }
```

Comment:
Maybe they are looking forward to learning fire spells, so they can finally burn things?